testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -129,6 +188,13 @@ Feature: Smoke tests for <client>
     And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed
     Then I should see a "This action's status is: Completed" text
 
+@slemicro53_minion
+@slemicro53_ssh_minion
+@slemicro52_minion
+@slemicro52_ssh_minion
+  Scenario: Check the configuration file afterwards
```

Comment:
So first we skip this and then after reboot we do it? Why not move the deployment whole scenario?

Reply:
Yes, that would make things easier.